### ISSUE TITLE
Mutual incompatibility with Rails 6.1.x over versions of `thor`

### ISSUE THREAD

**Issue by johncalvinyoung** [kytrinyx/approvals#92]
```
Bundler could not find compatible versions for gem "thor":
  In Gemfile:
    approvals (~> 0.0.24) was resolved to 0.0.24, which depends on
      thor (~> 0.18)

    factory_bot_rails was resolved to 6.2.0, which depends on
      railties (>= 5.0.0) was resolved to 6.1.3.2, which depends on
        thor (~> 1.0)
```

Any chance approvals can be updated to permit newer versions of `thor`?

**Comment by geeksam** [comment]
Did this in #90 but haven't got time to cut a release yet

**Comment by geeksam** [comment]
Just released 0.0.25 with updated thor dependency.
